### PR TITLE
Added ... ellipses in Menu Picks texts that invoke dialog 

### PR DESCRIPTION
**Pull request by zaryabkhan-rs**
> ### Motivate of the pull request
> - [ ] [To follow a UI Standard. Changes are made for this issue: <issue id>](https://github.com/os-fpga/FOEDAG/issues/465)
> - [ ] While menu commands are used for immediate actions, more information might be needed to perform the action. Indicate a command that needs additional information (including a confirmation) by adding an ellipses "..." at the end of the label.

> ### Describe the technical details
> - [ ]  UI standards call for all menu picks that require additional user input to have ... at the end of their text.

> 
> 
> 
> 
> 
>
> #### What does this pull request change?
> 
> 
> 
> - [ ] Changed the menu picks from "New" , "Open File ", "New Project", Open Project" to "New...", "Open File...", "New Project...", "Open Project..." and vice versa.
> 

> ### Which part of the code base require a change
> 
> - [ ] Menu Picks That Invoke Dialog

> ### Impact of the pull request

> - [ ] To have standardized form of Menu Picks for user actions.

### CODE REVIEW

File: src/NewProject/source_grid.cpp [hunk 1]
```diff
@@ -14,12 +14,12 @@ using namespace FOEDAG;
...
   m_lisFileData.clear();
-  m_btnAddFile = new QPushButton(tr("AddFile"), this);
+  m_btnAddFile = new QPushButton(tr("AddFile..."), this);
   connect(m_btnAddFile, &QPushButton::clicked, this, &sourceGrid::AddFiles);
-  m_btnAddDri = new QPushButton(tr("AddDir"), this);
+  m_btnAddDri = new QPushButton(tr("AddDir..."), this);
   connect(m_btnAddDri, &QPushButton::clicked, this,
           &sourceGrid::AddDirectories);
-  m_btnCreateFile = new QPushButton(tr("CreateFile"), this);
+  m_btnCreateFile = new QPushButton(tr("CreateFile..."), this);
```
Comment (ravic-rs) [inline src/NewProject/source_grid.cpp:22]: Can you add a screenshot for this change ?

Comment (zaryabkhan-rs) [reply src/NewProject/source_grid.cpp:22]: rew

**Comment by ravic-rs** [comment]
@zaryabkhan-rs please resolve the conflict. It came from this change https://github.com/os-fpga/FOEDAG/pull/542

File: src/NewProject/source_grid.cpp (hunk 1 above)
Comment (zaryabkhan-rs) [reply src/NewProject/source_grid.cpp:22]: > @zaryabkhan-rs the screenshot is outdated after the previous PR "Move Up" and "Move Down" change from previous PR (#542). "AddFile..." -> "Add File..." "AddDir..." -> "Add Directory..." "CreateFile..." -> "Create File..." FYI: @skyler-rs

As It was a separate issue and checkout from main, that's why those changes aren't merged with this PR. It should merge with #542 once this PR is merged.

Comment (zaryabkhan-rs) [reply src/NewProject/source_grid.cpp:22]: > @ravic-rs rew

Updated Screenshot!
dir

Comment (zaryabkhan-rs) [reply src/NewProject/source_grid.cpp]: @ravic-rs  Updated Screenshot ! 
upd

Comment (ravic-rs) [reply src/NewProject/source_grid.cpp]: Thanks!

**Review by ravic-rs — APPROVED** [review]
Looks good.

**Pull request merged**